### DISCUSSION TITLE
How dask is efficient in merging csv files.

### DISCUSSION THREAD

**Discussion by ggittu** [dask/dask#7408] [Q&A]
I tried to take it a traditional python way 

```
all_files = glob.glob("*.csv")
li = []
for filename in all_files:
    df = pd.read_csv(filename, index_col=None, header=0)
    li.append(df)
frame = pd.concat(li, axis=0, ignore_index=True)
frame.to_csv("entire.csv")
```

But same time I tried in dask way 
```
import dask.dataframe as dd
df = dd.read_csv('*.csv')
df.to_csv("entire.csv")
```

Both of them takes almost same time. Is this like this that dask and the normal python way behaves the same way on performance ? Pretty new to dask ,looking to hear your valuable suggestions.

**Comment by MrPowers** [comment]
@ggittu - can you try this and see if it improves performance?

```python
df.to_csv('/path/to/myfiles.csv', single_file = True)
```

See [here](https://stackoverflow.com/a/39573112) for more info.
